api.js

```diff
@@ -0,0 +1,147 @@
+const Joi = require('@hapi/joi');
+const Boom = require('@hapi/boom');
+const get = require('lodash/get');
+const flatten = require('lodash/flatten');
+
+module.exports = {
+    name: '@datawrapper/plugin-oembed',
+    version: '1.0.0',
+    options: {
+        routes: {
+            prefix: '/oembed'
+        }
+    },
+    register: (server, options) => {
+        const { events, event } = server.app;
+        const { models } = options;
+        const { Chart } = models;
+
+        // register new event type
+        event.GET_PUBLISHED_URL_PATTERN = 'GET_PUBLISHED_URL_PATTERN';
+
+        const { chartDomain } = server.methods.config('general');
```

Comment:
Is this a new configuration? I don't see it in the schema or on my local, only `plugins.publish-cloud.hostname`

Reply:
It is used for publishing to the local filesystem, but there it's called `chart_domain` currrently.

https://github.com/datawrapper/api/blob/master/src/server.js#L289

Reply:
it is a new configuration. since we already have `general.imageDomain`I would prefer `general.chartDomain`

Reply:
wait, why is it called `frontend.img_domain` now. we need to clean up our config files...

Reply:
I resolved this by using `plugins.publish-cloud.hostname` instead of `general.chartDomain`. since publishing to cloud providers is not a core feature it shouldn't be a core config.